components/calendar/src/dangi.rs

```diff
@@ -439,7 +439,8 @@ impl Dangi {
     ) -> FormattableYear {
         let era = Era(tinystr!(16, "dangi"));
         let number = year;
-        let cyclic = u8::try_from(div_rem_euclid64(number as i64 - 1 + 364, 60).1 as i32 + 1).unwrap_or(0);
+        let cyclic =
+            u8::try_from(div_rem_euclid64(number as i64 - 1 + 364, 60).1 as i32 + 1).unwrap_or(0);
```

Comment:
Anything `rem_euclid(60)` fits into a `u8`, so you can cast instead of using fallible conversions.

Also while you're here, please clean up the `let number = year`, either make the argument be `number`, or use `year` as the variable.